### PR TITLE
feat(config): add Defaro DAT-101 and a fingerprint for Fibaro FGFS101

### PR DESCRIPTION
**Pull request by saintman23**
PR description here

### CODE REVIEW

File: packages/config/config/devices/0x032e/dat-101.json [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+			"label": "State changes",
+			"description": "State changes -- general",
+			"maxNodes": 10,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External temperature sensor",
+			"description": "A group of associations to connect external room temperature sensor",
+			"maxNodes": 1
+		}
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: Please remove both descriptions

File: packages/config/config/devices/0x032e/dat-101.json [hunk 2]
```diff
@@ -0,0 +1,402 @@
+{
+	"manufacturer": "DEFARO",
+	"manufacturerId": "0x032e",
+	"label": "DAT-101",
+	"description": "Thermostat actuator DAT-101",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
	"description": "Thermostat Actuator",
```

File: packages/config/config/devices/0x032e/dat-101.json [hunk 3]
```diff
@@ -0,0 +1,402 @@
+{
+	"manufacturer": "DEFARO",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
	"manufacturer": "Defaro",
```
or is the manufacturer name really written in uppercase?

File: packages/config/config/devices/0x032e/dat-101.json [hunk 4]
```diff
@@ -0,0 +1,402 @@
...
+			"maxNodes": 10,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "External temperature sensor",
+			"description": "A group of associations to connect external room temperature sensor",
+			"maxNodes": 1
+		}
+	},
+	"paramInformation": [
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json:27]: Please read the [style guide](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide) thoroughly and apply it here.

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x032e/dat-101.json (hunk 3 above)
Comment (saintman23) [reply packages/config/config/devices/0x032e/dat-101.json]: I just copied DAS-102 code

Comment (AlCalzone) [reply packages/config/config/devices/0x032e/dat-101.json]: Its likely that we didn't clean up that file yet - its a pretty manual process.

File: packages/config/config/devices/0x032e/dat-101.json (hunk 4 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x032e/dat-101.json:27]: This is not resolved, let me highlight a few examples.

File: packages/config/config/devices/0x032e/dat-101.json [hunk 5]
```diff
@@ -0,0 +1,400 @@
...
+		},
+		"2": {
+			"label": "External temperature sensor",
+			"maxNodes": 1
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Relay mode",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: Labels should be in **Title Case**
https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=labels

File: packages/config/config/devices/0x032e/dat-101.json [hunk 6]
```diff
@@ -0,0 +1,400 @@
...
+			"description": "Selects NO/NC modes",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=parameter-descriptions
> Only include a description if it is necessary, helpful, and adds significant value.

This does not add any value over the options defined below

File: packages/config/config/devices/0x032e/dat-101.json [hunk 7]
```diff
@@ -0,0 +1,400 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal OPEN (default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: Don't mention `(default)` in labels, this is redundant with the `default` property

File: packages/config/config/devices/0x032e/dat-101.json [hunk 8]
```diff
@@ -0,0 +1,400 @@
...
+				{
+					"label": "Normal CLOSE",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "ΔT",
+			"description": "How much degrees should change the temperature to turn on / off the heating mode (hereinafter, ΔT), °С",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
			"label": "Hysteresis",
			"unit": "°С",
```

File: packages/config/config/devices/0x032e/dat-101.json [hunk 9]
```diff
@@ -0,0 +1,400 @@
...
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 7,
+			"defaultValue": 1,
+			"unsigned": true
+		},
+		{
+			"#": "3",
+			"label": "ΔT calculating",
+			"description": "Method for calculating on / off heating mode",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
			"label": "Hysteresis Offset",
```

File: packages/config/config/devices/0x032e/dat-101.json [hunk 10]
```diff
@@ -0,0 +1,400 @@
...
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 2,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Turning on the heating mode at room temperature, turning off at room temperature + ΔT",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
					"label": "Turn on at setpoint, turn off at setpoint + hysteresis",
```

File: packages/config/config/devices/0x032e/dat-101.json [hunk 11]
```diff
@@ -0,0 +1,400 @@
...
+					"value": 0
+				},
+				{
+					"label": "Turning on the heating mode at room temperature - ΔT, turning off at room temperature",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
					"label": "Turn on at setpoint - hysteresis, turn off at setpoint",
```

File: packages/config/config/devices/0x032e/dat-101.json [hunk 12]
```diff
@@ -0,0 +1,400 @@
...
+					"value": 1
+				},
+				{
+					"label": "Turning on the heating mode at room temperature - ½ ΔT, turning off at room temperature + ½ ΔT (default)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]:
```suggestion
					"label": "Turn on at setpoint - ½ hysteresis, turn off at setpoint + ½ hysteresis",
```

File: packages/config/config/devices/0x032e/dat-101.json (hunk 10 above)
Comment (AlCalzone) [reply packages/config/config/devices/0x032e/dat-101.json]: > Shorten labels wherever possible
> Labels should be clear and concise

https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=labels

File: packages/config/config/devices/0x032e/dat-101.json [hunk 13]
```diff
@@ -0,0 +1,400 @@
...
+			"description": "After the specified time, the lack of data from the room sensor will be considered a failure.",
+			"valueSize": 2,
+			"minValue": 100,
+			"maxValue": 30000,
+			"defaultValue": 30000,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "Minimal temperature, °С",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: `°C` needs to go into the `"unit"` property

File: packages/config/config/devices/0x032e/dat-101.json [hunk 14]
```diff
@@ -0,0 +1,400 @@
...
+			"description": "The user-set temperature cannot be lower than this value.",
+			"valueSize": 2,
+			"minValue": -127,
+			"maxValue": 127,
+			"defaultValue": 4,
+			"unsigned": true
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: This is definitely not `unsigned`.

File: packages/config/config/devices/0x032e/dat-101.json [hunk 15]
```diff
@@ -0,0 +1,400 @@
...
+			"valueSize": 2,
+			"minValue": 100,
+			"maxValue": 30000,
+			"defaultValue": 30000,
+			"unsigned": true
+		},
+		{
+			"#": "6",
+			"label": "Minimal temperature, °С",
+			"description": "The user-set temperature cannot be lower than this value.",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x032e/dat-101.json]: > unnecessary descriptions that merely restate the label must be removed. Additionally, information like units or available ranges should be removed as that information is provided to UIs through other properties.

https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide?id=parameter-descriptions

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Et cetera... Many descriptions are unnecessary or can be avoided by rewording the labels.

Again, please read the [style guide](https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide) thoroughly and apply it here.

**Comment by saintman23** [comment]
@zwave-js-bot fix lint

**Comment by AlCalzone** [comment]
@zwave-js-bot fix lint

**Review by AlCalzone — APPROVED** [review]
Thanks - I had a few more things, but I did them myself.

**Comment by saintman23** [comment]
> Thanks - I had a few more things, but I did them myself.

Thanks a lot for your help.

**Pull request merged**